.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
 .next/
-.env
\ No newline at end of file
+.env
+tsconfig.tsbuildinfo
```

Comment:
new lines :/

Reply:
I plan to refactor this later with segregations between what's ignored for what.